Makefile

```diff
@@ -84,6 +84,9 @@ bin/unpack:
 build-local-container: ## Builds provisioner container image locally
 	docker build -f Dockerfile -t $(IMAGE) .
 
+build-container: build
```

Comment:
Should we swap the build-container and the build-local-container target names, i.e. the target that's responsible for building the Dockerfile.local dockerfile is named build-local-container?